src/Command/Ide/IdeCreateCommand.php

```diff
@@ -96,7 +101,7 @@ protected function waitForDnsPropagation($ide_url): void {
         $this->logger->debug($e->getMessage());
       }
     });
-    LoopHelper::addTimeoutToLoop($loop, 15, $spinner, $this->output);
+    LoopHelper::addTimeoutToLoop($loop, 20, $spinner, $this->output);
```

Comment:
Why 20 minutes? I'm not aware of the average but wouldn't 15 suffice?

Reply:
15mn often fails for people not using Google or Cloudflare DNS servers. 20 is just buying us extra time, but won't fix the issue with DNS propagation times in Australia, India and parts of EMEA.

Reply:
That's good context to know, thanks.